pyracing/helpers.py

```diff
@@ -30,6 +30,7 @@ def now_five_min_floor():
 
 
 def parse_encode(string):
-    if not string:
+    if not type(string) is str:
         return None
-    return urllib.parse.unquote(string).replace('+', ' ')
+    else:
```

Comment:
No need for an else here, just put the return outside of the if. It reads better without unnecessary nesting.

Reply:
```suggestion
    
```
So this line can go completely...